easybuild/easyconfigs/p/ProBiS/ProBiS-20230403-gompi-2022b.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'ProBiS'
+version = '20230403'
+local_commit = 'f10a9f4c'
+
+homepage = 'http://insilab.org/probis-algorithm'
+description = """ProBiS algorithm aligns and superimposes complete protein surfaces,
+surface motifs, or protein binding sites."""
+
+toolchain = {'name': 'gompi', 'version': '2022b'}
+
+source_urls = ['https://gitlab.com/janezkonc/probis/-/archive/master/']
+sources = ['%s.tar.gz' % local_commit]
+checksums = ['5f81df7498daefe0e66082561f31d4c66f75cf1eebbed6430e3d78dd3097896b']
+
+dependencies = [
+    ('GSL', '2.7'),
+]
+
+buildopts = 'probis CC="mpic++" CFLAGS="-std=c++11 -O3 -Wall -Wno-unused-function -DNDEBUG -pthread"'
```

avoid hardcoding `mpic++` and `-std=c++11 -O3`:

```suggestion
buildopts = 'probis CC="$MPICXX" CFLAGS="$CFLAGS -Wall -Wno-unused-function -DNDEBUG -pthread"'
```